src/optimistic-oracle-v2/implementation/ManagedOptimisticOracleV2.sol

```diff
@@ -25,6 +25,18 @@ contract ManagedOptimisticOracleV2 is ManagedOptimisticOracleV2Interface, Optimi
         BondRange range;
     }
 
+    struct CustomBond {
+        uint256 amount;
+        // isSet is not used anymore as amount cannot be set to 0, but it is kept for storage layout compatibility.
+        bool isSet;
```

Comment:
Can we name it something distinct so it's clear it should not be used?

`DEPRECATED_isSet;`, for example.

See [here](https://github.com/across-protocol/contracts/blob/f2415a29af7aadd61321d26da51d14a233ec2db8/contracts/SpokePool.sol#L63) as an example.

Reply:
The upgrade verification check would fail and it is not possible to tag renamed struct member. The other alternative is to use `@custom:oz-retyped-from`, but that would forego storage layout checks for the whole struct.